components/resources/quizzes/langQuiz/LangResults.tsx

```diff
@@ -3,22 +3,70 @@ import SkillifyNavbar from "../shared/SkillifyNavbar";
 
 type LangResultsProps = {
   onBackClick: () => void;
+  score: { [key: string]: number };
 };
 
-const LangResults = ({ onBackClick }: LangResultsProps) => {
+const LangResults = ({ onBackClick, score }: LangResultsProps) => {
+  const maxLangScore = (score: { [key: string]: number }) => {
+    let maxScore = 0;
+    let preferredLang = "";
+    for (const [key, value] of Object.entries(score)) {
+      if (value > maxScore) {
+        maxScore = value;
+        preferredLang = key;
+      }
+    }
+    return preferredLang;
+  };
+
+  const preferredLang = maxLangScore(score);
+
+  type resultData = {
+    body: string;
+    src: string;
+    alt: string;
+    lang: string;
+    size: string;
+  };
+
+  const results: { [key: string]: resultData } = {
+    JavaScript: {
+      body: "The first coding language you should learn is...",
+      src: "/images/quiz/languages-quiz/javascript.png",
+      alt: "JavaScript",
+      lang: "JavaScript",
+      size: "scale-75",
+    },
+    Python: {
+      body: "The first coding language you should learn is...",
+      src: "/images/quiz/languages-quiz/python.png",
+      alt: "Python",
+      lang: "Python",
+      size: "scale-50 h-80",
```

Comment:
the size of the image is view logic, but the rest of this object is business logic. Business logic should go in the api folder, but that means these sizes would also have to go into the api folder, which is not ideal.

Is it possible to use scale-75 for all images, and avoid having to store this information in the results object? Can you use a different image for python so we can use scale-75 everywhere